test/PoolFactory/deployPool.spec.ts

```diff
@@ -265,4 +270,127 @@ describe("PoolFactory.deployPool", () => {
+            ).toNumber()
+            const newFeeSplit = (
+                await newPool.secondaryFeeSplitPercent()
+            ).toNumber()
+            expect(oldFeeSplit).to.eq(10)
+            expect(newFeeSplit).to.eq(20)
+        })
+
+        it("pool fee transfers new fee split percentage", async () => {
+            const deploymentData = {
+                poolName: POOL_CODE_2,
+                frontRunningInterval: 3,
+                updateInterval: 5,
+                leverageAmount: 2,
+                quoteToken: token.address,
+                oracleWrapper: oracleWrapper.address,
+                settlementEthOracle: settlementEthOracle.address,
+            }
+
+            const poolNumber = (await factory.numPools()).toNumber()
+            await factory.deployPool(deploymentData)
+            const newPoolAddress = await factory.pools(poolNumber)
+            const newPool = (await ethers.getContractAt(
+                "LeveragedPool",
+                newPoolAddress
+            )) as LeveragedPool
+
+            let commiter = await newPool.poolCommitter()
+            const poolCommitter = (await ethers.getContractAt(
+                "PoolCommitter",
+                commiter
+            )) as PoolCommitter
+
+            // Make commits
+            await token.approve(
+                newPool.address,
+                ethers.utils.parseEther("10000")
+            )
+            await createCommit(
+                poolCommitter,
+                LONG_MINT,
+                ethers.utils.parseEther("2000")
+            )
+            await createCommit(
+                poolCommitter,
+                SHORT_MINT,
+                ethers.utils.parseEther("2000")
+            )
+            const signers = await ethers.getSigners()
+            await newPool.setKeeper(signers[0].address)
+            newPool.updateSecondaryFeeAddress(secondFeeAddress)
+
+            // Execute commits no price change
+            await timeout(updateInterval * 1000)
+            const lastPrice = ethers.utils.parseEther(
+                getRandomInt(99999999, 1).toString()
+            )
+            await newPool.poolUpkeep(lastPrice, lastPrice)
+
+            // Upkeep pool with price change
+            await timeout(updateInterval * 1000)
+            await newPool.poolUpkeep(
+                lastPrice,
+                BigNumber.from("2").mul(lastPrice)
+            )
+
+            let feesPaidPrimary = await token.balanceOf(feeAddress)
+            let feesPaidSecondary = await token.balanceOf(secondFeeAddress)
+            expect(
+                parseFloat(ethers.utils.formatEther(feesPaidPrimary)) /
+                    parseFloat(ethers.utils.formatEther(feesPaidSecondary))
+            ).to.eq(4)
+        })
+    })
+
+    context("When secondary fee split is changed too high", async () => {
+        it("change fee split to 100 should revert error", async () => {
```

Comment:
```suggestion
        it("Reverts", async () => {
```

I think the `context` captures the fact that the secondary fee split is being modified to an excessive value.